code/modules/mob/living/silicon/robot/death.dm

```diff
@@ -8,8 +8,9 @@
 	if(camera)
 		camera.status = 0
 	if(module)
-		var/obj/item/weapon/gripper/G = locate(/obj/item/weapon/gripper) in module
-		if(G) G.drop_item()
+		for (var/obj/item/weapon/gripper/G in module)
+			if(G)
```

Comment:
This is already checked in the previous line.

Reply:
doing